routes/matches/matches.js

```diff
@@ -329,14 +329,19 @@ router.get("/", async (req, res, next) => {
     let isAscending = req.query?.asc == null ? false : req.query.asc;
     let sql = 
       "SELECT mtch.id, mtch.user_id, mtch.server_id, mtch.team1_id, mtch.team2_id, mtch.winner, mtch.team1_score, " +
-      "mtch.team2_score, mtch.team1_series_score, mtch.team2_series_score, mtch.team1_string, mtch.team2_string, " +
+      "mtch.team2_score, mtch.team1_series_score, mtch.team2_series_score, " +
       "mtch.cancelled, mtch.forfeit, mtch.start_time, mtch.end_time, mtch.max_maps, mtch.title, mtch.skip_veto, mtch.private_match, " +
-      "mtch.enforce_teams, mtch.min_player_ready, mtch.season_id, mtch.is_pug, usr.name as owner, mp.team1_score as team1_mapscore, mp.team2_score as team2_mapscore " +
-      "FROM `match` mtch JOIN user usr ON mtch.user_id = usr.id LEFT JOIN map_stats mp ON mp.match_id = mtch.id " +
-      "WHERE cancelled = 0 " +
-      "OR cancelled IS NULL " +
+      "mtch.enforce_teams, mtch.min_player_ready, mtch.season_id, mtch.is_pug, usr.name as owner, mp.team1_score as team1_mapscore, mp.team2_score as team2_mapscore, " +
+      "gs.ip_string, gs.port, gs.gotv_port, gs.display_name, mtch.team1_string, mtch.team2_string " +
```

Comment:
I'm still against this query here. I don't want to be giving out game IP strings through the API without the user being authenticated. Please either break this out into an individual query (check if req.user exists, and is a part of the match/super_admin) or try not to include the IP. This gives people way too easy of a time to just DDoS a game server in drive-by attacks.